Dockerfile

```diff
@@ -24,4 +34,13 @@ RUN set -e \
  && SECRET_KEY=temp HOOVER_DB='postgresql://search:search@search-pg:5432/search' ./manage.py collectstatic --noinput \
  && chmod +x /wait
 
-CMD ./runserver
+ENV USER_NAME $UNAME
+ENV DATA_DIR /opt/hoover/search
+
+mkdir -p /opt/hoover/metrics
+chown -R 666:666 /opt/hoover/metrics
```

Comment:
don't do `chown -R` here - this dir is empty, you just created it, so be explicit about it

also, use $UID $GID instead of the numbers